OmniKit/Model/FaultEventCode.swift

```diff
@@ -120,194 +175,304 @@ public struct FaultEventCode: CustomStringConvertible {
         if let faultType = faultType {
             switch faultType {
             case .noFaults:
-                return LocalizedString("Fault Event Type 0x00: No faults, system running normally.", comment: "Fault Event Type 0x00: No faults, system running normally.")
+                return LocalizedString("Fault Event Type 0x00: No faults, system running normally", comment: "Fault Event Type 0x00: No faults, system running normally")
```

Comment:
If you want to prefix *all* faults with "Fault Event Type 0xxx", you can split that out into a prefix generated by the FaultEventCode.description function.